### PR TITLE
Enhance : Update readme with new content

### PR DESCRIPTION
**Pull request by cp-jagruti-a**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [ ] Add MR description if applicable
- [ ] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [ ] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [ ] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [ ] Make MR mark as ready before submitting it for review
- [ ] Add images, videos, or gifs of UI
- [ ] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**

- [ ] Inspector (all iPhones and android devices like samsung, motorola)
- [ ] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook (chrome, firefox, safari)
- [ ] iPhone(safari)
- [ ] Available android devices (chrome, firefox)
- [ ] iPad(safari)
- [ ] Tablet (chrome, firefox)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,65 @@
 <h1 align="left"><strong>Repository of canopas website includes intuitive UI designs</strong></h1>
```
Comment (eng2-veristicky) [inline README.md:1]: Replace this with `Website with intuitive UI designs`

File: README.md [hunk 2]
```diff
@@ -1,51 +1,65 @@
...
-Source code repository of [Canopas Software Website](https://canopas.com).
-
 We intend to keep this open source. Plan is to keep the repository up to date with latest technologies and best practices.
 
-## Requirements
+## Showcase
 
-- Node18
-- Go 1.20
+This repository contains the working code of [our website](https://canopas.com). You can check out the link to view a live example of this repository.
```
Comment (eng2-veristicky) [inline README.md:7]: our website -> canopas website

File: README.md [hunk 3]
```diff
@@ -1,51 +1,65 @@
...
 
-  ```
-  yarn build
-  ```
+# Key Features
 
-  ```
-  yarn serve
-  ```
+1. **Vuejs in Website:** Our website is built using Vue.js, a progressive JavaScript framework that offers a flexible and efficient way to build user interfaces. Vue.js is renowned for its simplicity and ease of integration, making it an ideal choice for creating dynamic and interactive web applications.
```
Comment (eng2-veristicky) [inline README.md:40]: Vuejs in Website -> Vuejs as frontend

Also include `Go as backend` point

File: README.md [hunk 4]
```diff
@@ -1,51 +1,65 @@
...
+2. **Server Side Rendering:** We have used SSR(Server Side Rendering) in website to improve performance. Server will render UI in `<!--app-html-->` part of `vue-frontend/index.html`. Due to this, reformating of `index.html` can cause UI breaking.
 
-You can access the page by pointing a web browser at http://localhost:8080.
+3. **SEO-friendly URLs and Metadata:** We prioritize SEO best practices to ensure that our website receive the visibility they deserve in search engines. From meta tags to URL structure and sitemaps, our canopas website is equipped with all the essential elements for optimal SEO.
```
Comment (eng2-veristicky) [inline README.md:43]: our canopas website -> canopas website

File: README.md [hunk 5]
```diff
@@ -1,51 +1,65 @@
...
+
+4. **Responsive and Mobile-friendly Design:** We understand the importance of a mobile-friendly design in modern SEO. Our canopas website is fully responsive and adapts seamlessly to various screen sizes, providing a positive user experience across devices.
+
+5. **Image Optimization:** Images play a significant role in SEO and user experience. Our project automatically optimizes images, reducing their file sizes without compromising quality, resulting in faster loading times and better search rankings.
```
Comment (eng2-veristicky) [inline README.md:47]: We are not doing it automatically. Remove this point

**Review by eng2-veristicky — CHANGES_REQUESTED** [review]

**Comment by eng2-veristicky** [comment]
Add mainly used dependencies for frontend and backend

Also add which AWS stack we are using for deployment

File: README.md
Comment (cp-sumi-k) [inline README.md:1]: Add mainly used dependencies for frontend and backend

File: README.md (hunk 1 above)
Comment (cp-sumi-k) [inline README.md:1]: Replace title with `Website with intuitive UI designs`

File: README.md (hunk 2 above)
Comment (cp-sumi-k) [inline README.md:7]: our website -> canopas website

File: README.md (hunk 3 above)
Comment (cp-sumi-k) [inline README.md:40]: Vuejs in Website -> Vuejs as frontend

Also add `Go as backend` point

File: README.md (hunk 5 above)
Comment (cp-sumi-k) [inline README.md:47]: We are not optimizing images automatically, remove this point

File: README.md (hunk 4 above)
Comment (cp-sumi-k) [inline README.md:43]: our canopas website -> canopas website

File: README.md [hunk 6]
```diff
@@ -1,51 +1,65 @@
...
+6. **CI/CD and Deployment:** With CI in place, every code change is automatically tested and integrated into the main codebase. This ensures that the main branch always remains stable, reducing the chances of bugs and allowing for rapid deployment.
```
Comment (cp-sumi-k) [inline README.md:48]: Add AWS stack we are using for deployment.

Like, 

```
This repository contains AWS ECR/ECS and EC2 stack for frontend and backend deployment.
```

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -1,51 +1,65 @@
...
 
-- Node18
-- Go 1.20
+This repository contains the working code of [our website](https://canopas.com). You can check out the link to view a live example of this repository.
 
-### Note :
+### Home Page
 
-- We have used SSR(Server Side Rendering) in website to improve performance. Server will render UI in `<!--app-html-->` part of `vue-frontend/index.html`. Due to this, reformating of `index.html` can cause UI breaking.
+![Home page](https://github.com/canopas/canopas-website/assets/85920585/735fbc89-6c9e-48c4-b41b-d962011f116e)
```
Comment (cp-mansi-d) [inline README.md:11]: convert into gif and then add

**Review by cp-mansi-d — DISMISSED** [review]

File: README.md [hunk 8]
```diff
@@ -115,15 +126,83 @@ You can access the page by pointing a web browser at http://localhost:8080.
...
+
+### [testify](https://github.com/stretchr/testify)
+
+- A popular testing toolkit for Go. It provides various assertion functions and testing utilities to enhance the readability and maintainability of your test code.
+
+### [cors](https://github.com/gin-contrib/cors)
+
+- A middleware for Gin that provides Cross-Origin Resource Sharing (CORS) support, allowing controlled access to resources hosted on different domains.
+
+### [gzip](https://github.com/gin-contrib/gzip)
```
Comment (cp-sumi-k) [inline README.md:201]: I think the font size is different for both backend and frontend dependencies

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**